server/src/main/java/org/apache/druid/client/indexing/ClientCompactionTaskGranularitySpec.java

```diff
@@ -22,26 +22,22 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import org.apache.druid.java.util.common.granularity.Granularity;
-import org.apache.druid.segment.indexing.granularity.BaseGranularitySpec;
 
 import java.util.Objects;
 
-public class ClientCompactionTaskQueryGranularitySpec
+public class ClientCompactionTaskGranularitySpec
```

Comment:
Can you add a javadoc for this please. Why is this needed? Why we haven't chosen to use the GranularitySpec class that already exists?

I'm ok if you do this in a follow up change.

Reply:
I'll add in a followup change